include/swift/SIL/MemAccessUtils.h

```diff
@@ -8,7 +8,26 @@
 // See https://swift.org/LICENSE.txt for license information
 // See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 //
-// These utilities live in SIL/ so they be used by SIL verification.
+// These utilities model formal memory access locations as marked by
```

Comment:
Can you make this match what LLVM does for file level comments? https://llvm.org/docs/CodingStandards.html#file-headers

Reply:
Yep